mono/metadata/image.c

```diff
@@ -1171,7 +1205,7 @@ MonoImage *
 mono_image_loaded_by_guid_full (const char *guid, gboolean refonly)
 {
 	GuidData data;
-	GHashTable *loaded_images = refonly ? loaded_images_refonly_hash : loaded_images_hash;
+	GHashTable *loaded_images = loaded_images_hash (TRUE, refonly);
```

Comment:
I would welcome any input on whether people think the interface to loaded_images_hash here is a good one.